source/slang/ir.cpp

```diff
@@ -3641,6 +3671,14 @@ namespace Slang
         // and their instructions.
         cloneFunctionCommon(context, clonedFunc, originalFunc);
 
+        //// for now, clone all unreferenced witness tables
```

Comment:
Delete this block since it isn't used? Or keep it around in case we need it later?